app/main.py

```diff
@@ -1,8 +1,16 @@
 class Person:
-    # write your code here
-    pass
+    dict_of_person_obj = []
+    def __init__(self, name: str, age: int):
+        self.name = name
+        self.age = age
+        Person.dict_of_person_obj[self.name] = self
 
-
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+def create_dict_of_person_obj(people: list) -> list:
+    list_person = []
+    for person in people:
+        list_person.append(Person(person.get("name"),person.get("age")))
+        if not isinstance(person.get('wife'), None):
+             person["wife"] = Person.dict_of_person_obj.get(person["name"])
+        if not isinstance(person.get('husband'), None):
+             person["husband"] = Person.dict_of_person_obj.get(person["name"])
+    return list_person
```

Comment:
![test](https://github.com/mate-academy/py-person-class/assets/98127979/99c3c96f-96c5-48d2-9c88-d914c992a27b)

Reply:
did you run tests before pushing? do not rename the create_person_list function